src/tui/editor.zig

```diff
@@ -2029,6 +2029,14 @@ pub const Editor = struct {
         return false;
     }
 
+    fn is_eol_vim(root: Buffer.Root, cursor: *const Cursor, metrics: Buffer.Metrics) bool {
```

Comment:
This is effectively the same as `is_eol_right_vim`. In fact, I think `is_eol_right_vim` is more general because it uses `>=`, so I think we should use that and drop this function.

Reply:
Ah, I get it now. Yes, you can leave this function in then.